credential_adapter/credential_adapter.go

```diff
@@ -30,6 +31,11 @@ type IssuanceChallenge struct {
 	Url      string
 }
 
+type PresentationChallenge struct {
```

Comment:
What does it do, can you pls comment it?

Reply:
What comment do you think would be valuable to put here? Or are you asking me to more generally explain what the purpose of the presentation challenge is (I think I'll put that in the scope either way though)

Reply:
@sKudryashov Based on the additions to the Notion scope document am I good to resolve this? If not what additions do you want here?

Reply:
Yes